src/brain/gocdNoDeploysAlert/index.ts

```diff
@@ -18,6 +18,7 @@ import {
 import { getLastGetSentryGoCDDeploy } from '@/utils/db/getLatestDeploy';
 
 const PIPELINE_FILTER = [
+  'deploy-getsentry-backend-s4s',
```

Comment:
Not a blocker, but is it worth putting this as a variable in `config/index.ts`?

Reply:
Will do this next time it is used